cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/IRBlock.qll

```diff
@@ -1,7 +1,7 @@
 private import internal.IRInternal
 import Instruction
 import semmle.code.cpp.ir.implementation.EdgeKind
-private import Construction::BlockConstruction
+import Cached
```

Comment:
We seem to have a convention for these "Cached" modules: the `import Cached` goes right before the module definition:
```
private import Cached
private cached module Cached {
…
}
```

Reply:
Also, regardless of where the `import` statement winds up, it should be `private import Cached`, since those predicates are used only within `IRBlock.qll`.